tests/testing_common.py

```diff
@@ -1417,11 +1414,24 @@ def _test_delete_adapter(self, model_id, config_cls, config_kwargs):
             for attr in attributes_to_check:
                 assert adapter_to_delete not in getattr(target, attr)
 
+        has_auxiliary = bool(getattr(config, "modules_to_save", False))
+        if has_auxiliary:
+            for module in model.modules():
+                if isinstance(module, AuxiliaryTrainingWrapper):
+                    assert adapter_to_delete not in module.modules_to_save
+                    assert module.active_adapters == ["default"]
+
```

Comment:
This looks like it tests auxiliary training wrappers but it only tests `modules_to_save`. Maybe the following would be more general?

```suggestion
        for module in model.modules():
            if isinstance(module, AuxiliaryTrainingWrapper):
                assert adapter_to_delete not in module._adapters
                assert module.active_adapters == ["default"]

```

Reply:
Yes, totally right. I changed the check and of course this revealed new bugs. Those should now be fixed too, but I'm sure they're not the last ones :-/